workspaces/js-x-ray/test/probes/isLiteral.spec.ts

```diff
@@ -78,50 +78,34 @@ test("should not throw any warnings without hexadecimal value (and should call a
   assert.strictEqual(astNode.value, "hello world!");
 });
 
-test("should detect shady link when an URL is bit.ly", () => {
-  const str = "const foo = 'http://bit.ly/foo'";
-  const ast = parseScript(str);
-  const sastAnalysis = getSastAnalysis(isLiteral).execute(ast.body);
-
-  assert.strictEqual(sastAnalysis.warnings().length, 1);
-  const warning = sastAnalysis.getWarning("shady-link");
-  assert.strictEqual(warning.value, "http://bit.ly/foo");
-});
-
-test("should detect shady link when an URL is ipinfo.io when protocol is http", () => {
-  const str = "const foo = 'http://ipinfo.io/json'";
-  const ast = parseScript(str);
-  const sastAnalysis = getSastAnalysis(isLiteral).execute(ast.body);
-  assert.strictEqual(sastAnalysis.warnings().length, 1);
-  const warning = sastAnalysis.getWarning("shady-link");
-  assert.strictEqual(warning.value, "http://ipinfo.io/json");
-});
-
-test("should detect shady link when an URL is ipinfo.io when protocol is https", () => {
-  const str = "const foo = 'https://ipinfo.io/json'";
-  const ast = parseScript(str);
-  const sastAnalysis = getSastAnalysis(isLiteral).execute(ast.body);
-  assert.strictEqual(sastAnalysis.warnings().length, 1);
-  const warning = sastAnalysis.getWarning("shady-link");
-  assert.strictEqual(warning.value, "https://ipinfo.io/json");
-});
-
-test("should detect shady link when an URL is httpbin.org when protocol is http", () => {
-  const str = "const foo = 'http://httpbin.org/ip'";
-  const ast = parseScript(str);
-  const sastAnalysis = getSastAnalysis(isLiteral).execute(ast.body);
-  assert.strictEqual(sastAnalysis.warnings().length, 1);
-  const warning = sastAnalysis.getWarning("shady-link");
-  assert.strictEqual(warning.value, "http://httpbin.org/ip");
-});
-
-test("should detect shady link when an URL is httpbin.org when protocol is https", () => {
-  const str = "const foo = 'https://httpbin.org/ip'";
-  const ast = parseScript(str);
-  const sastAnalysis = getSastAnalysis(isLiteral).execute(ast.body);
-  assert.strictEqual(sastAnalysis.warnings().length, 1);
-  const warning = sastAnalysis.getWarning("shady-link");
-  assert.strictEqual(warning.value, "https://httpbin.org/ip");
+describe("known suspicious domain", () => {
+  // CONSTANTS
+  const kKnownSuspiciousDomains = ["bit.ly/foo",
+    "ipinfo.io/json",
+    "httpbin.org/ip",
+    "api.ipify.org/ip"];
```

Comment:
If not at the top level no need to be a CONSTANT (here that's just a variable inside a test)

```suggestion
  const suspiciousDomains = [
    "bit.ly/foo",
    "ipinfo.io/json",
    "httpbin.org/ip",
    "api.ipify.org/ip"
  ];
```